docs/user/alerting/action-types/pagerduty.asciidoc

```diff
@@ -4,19 +4,142 @@
+
+* You must have
+application privileges to access Metrics, APM, Uptime, or SIEM.
+* If you are using a self-managed deployment with security, you must have
+Transport Security Layer (TLS) enabled for communication <<configuring-tls-kib-es, between Elasticsearch and Kibana>>.
+Elastic Alerts uses API keys to secure background alert checks and actions,
+and API keys require {ref}/configuring-tls.html#tls-http[TLS on the HTTP interface].
+
+Although not a requirement, to harden the integrations security you might want to
+review the <<action-settings, Actions settings>> that are available to you.
+
+[float]
+[[pagerduty-support]]
+==== Support
+If you need help with this integration, get in touch with the Kibana team by visiting
+https://support.elastic.co[support.elastic.co] or by using the *Ask Elastic* option in the Kibana Help menu.
+You can also select the Kibana category at https://discuss.elastic.co/[discuss.elastic.co].
+
+[float]
+[[pagerduty-integration-walkthrough]]
+==== Integration with PagerDuty walkthrough
+
+[float]
+[[pagerduty-in-pagerduty]]
+===== In PagerDuty
+
+. From the *Configuration* menu, select *Services*.
+. Add an integration to a service:
++
+* If you are adding your integration to an existing service,
+click the name of the service you want to add the integration to.
+Then, select the *Integrations* tab and click the *New Integration* button.
+* If you are creating a new service for your integration,
+go to
+https://support.pagerduty.com/docs/services-and-integrations#section-configuring-services-and-integrations[Configuring Services and Integrations]
+and follow the steps outlined in the *Create a New Service* section, selecting *Elastic* as the *Integration Type* in step 4.
+Continue with the <<pagerduty-in-elastic, In Elastic>> section once you have finished these steps.
+
+. Enter an *Integration Name* in the format Elastic-service-name (for example, Elastic-Alerting or Kibana-APM-Alerting)
+and select Elastic from the *Integration Type* menu.
+. Click *Add Integration* to save your new integration.
++
+You will be redirected to the *Integrations* tab for your service. An Integration Key is generated on this screen.
++
+[role="screenshot"]
+image::user/alerting/images/pagerduty-integration.png[PagerDuty Integrations tab]
+
+. Save this key, as you will use it when you configure the integration with Elastic in the next section.
+
+[float]
+[[pagerduty-in-elastic]]
+===== In Elastic
+
+. Create a PagerDuty Connector in Kibana.  You can:
++
+* Create a connector as part of creating an alert by selecting PagerDuty in the *Actions*
+section of the alert configuration and selecting *Add new*.
+* Alternatively, create a connector by navigating to *Management* from the {kib} navbar and selecting
+*Alerts and Actions*. Then, select the *Connectors* tab, click the *Create connector* button, and select the PagerDuty option.
+
+. Configure the connector by giving it a name and entering the API URL and Routing Key.
```

Comment:
I've learned that the API URL is optional, so it might be worth calling out here.

`Configure the connector by giving it a name, using the default or optional API URL and Routing Key.`

@arisonl @gchaps probably not the best phrasing, but does it make sense to call out that it's optional without having to click into the config docs?  I actually opened an issue for this to fix in product: https://github.com/elastic/kibana/issues/63768

Reply:
@alexfrancoeur are both the API URL and Routing Key optional, or just the API URL?